project-addons/purchase_proposal/product.py

```diff
@@ -62,8 +63,8 @@ def compute_last_sixty_days_sales(self, records=False):
                 moves = move_obj.search([('date', '>=', days_data[0]),
                                          ('state', '=', 'done'),
                                          ('product_id', '=', product.id),
-                                         ('picking_type_code', '=',
-                                          'outgoing'),
+                                         ('picking_type_id', '=',
```

Comment:
Mejor picking_type_id.code = 'outgoing', o para que sea más rápida (evitamos un join) haces dos, primero obtiene los ids de los picking_type con code = 'outgpoing luego haces un picking_type_id in IDS